it/config/src/main/content/jcr_root/apps/system/config/com.adobe.granite.toggle.impl.dev.DynamicToggleProviderImpl.cfg.json

```diff
@@ -21,6 +21,7 @@
     "FT_SITES-19631",
     "FT_FORMS-14255",
     "FT_FORMS-14068",
-    "FT_FORMS-16351"
```

Comment:
FORMS-2447 is already closed, we should not be using an FT of a closed ticket. Please use the Epic/Story ID which is specific to this ticket.

Reply:
Sure, @iamsudhanshu I will update the FT. Just wanted to confirm if we indeed to keep this feature behind FT for AFv2.

Reply:
All features have to behind an FT - including this.